src/transformers/__init__.py

```diff
@@ -1917,6 +1932,18 @@
             MBartForSequenceClassification,
             MBartModel,
         )
+        from .models.megatron_bert import (
+            MEGATRON_BERT_PRETRAINED_MODEL_ARCHIVE_LIST,
+            MegatronBertForMaskedLM,
+            MegatronBertForMultipleChoice,
+            MegatronBertForNextSentencePrediction,
+            MegatronBertForPreTraining,
+            MegatronBertForQuestionAnswering,
+            MegatronBertForSequenceClassification,
+            MegatronBertForTokenClassification,
+            MegatronBertLayer,
```

Comment:
This one is note added in the dictionary above. It should either be removed here or added up there.

Reply:
Addressed in commit f155bc48ffd4606aab7461035e49c0318e57fd9c